src/assets/tech.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "id": "SHOVEL",
+    "name": "Shovels and Diggers",
+    "baseCost": 100,
+    "dependTech": null,
+    "dependLevel": 0
+  },
+  {
+    "id": "HAMMER",
+    "name": "Hammers and Picks",
+    "baseCost": 500,
+    "dependTech": "SHOVEL",
```

Comment:
if this was an array you can have combos

Reply:
the tech dependencies? i _suppose_ i could make it an array. i don't have any plans right now for tools that depend on multiple technologies, but the whole reason i made this tool editor in the first place was to make adding new tools in the future easier. the only downside would be i'd have to change the tech dependency field in the editor to like a multi-select box or something.

Reply:
welp found [this](https://vue-multiselect.js.org) in a quick search so i guess i'm doing multi-tool dependencies now

Reply:
tools and tech can now have multiple dependencies. wound up hand-rolling a component to handle multiple tech dependencies on tools because i forgot i have minimum level requirements...